src/main/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceService.java

```diff
@@ -39,6 +40,23 @@ public ObjectResponse create(String accountUrn, ObjectCreate createObject) {
     }
 
     @Override
+    public Optional<ObjectResponse> update(String accountUrn, ObjectResponse updateObject) {
```

Comment:
I do not think we want to be updating an ObjectResponse back to the server.  This should be an ObjectUpdate with the proper fields.  An ObjectResponse is only going to be returned from the server to the requester.